docs/proposals/label-associated-quota.md

```diff
@@ -0,0 +1,72 @@
+# Label Constrained Quota
+
+## Problem
+Cluster-administrators want to allot a total quota cap to a particular entity (users to start with) and allow that entity to allocate the quota amongst "his" projects as he sees fit.
+Our mechanism for relating entities is label selection, so we can create a `ResourceQuotaAllocation` object at the cluster scope that applies allotment totals against `ResourceQuota`
+objects.  By associating projects to owners using structured labels, it becomes possible to allocate quota the projects "owned" by a given user.
+
+
+## Types
+```go
+type ResourceQuotaAllocation struct{
+	unversioned.TypeMeta
+	kapi.ObjectMeta
+
+	Spec ResourceQuotaAllocationSpec
+	Status ResourceQuotaAllocationStatus
+}
+
+type ResourceQuotaAllocationSpec struct{
+	// Selector is used to find the set of projects that this ResourceQuotaAllocation is for
+	Selector extensions.LabelSelector
+
+	// Hard is the description of the limits
+	Hard kapi.ResourceQuotaSpec
```

Comment:
Since `Hard` is a member field of `ResourceQuotaSpec`, I think it will be a little confusing to see `Hard.Hard` in the code base...  Is this more of a quota template?  Can we call it `Template` or something else instead?